kolibri/plugins/perseus_viewer/assets/src/views/PerseusRendererIndex.vue

```diff
@@ -124,6 +124,67 @@
 
   const blobImageRegex = /blob:[^)^"]+/g;
 
+  Khan.imageUrls = {};
+
+  function getImagePaths(itemResponse) {
+    const graphieMatches = {};
```

Comment:
What's `graphie`? Is this a perseus specific image term?

Reply:
Yes - it allows loading of either a JSON blob or an SVG file to display graph style images.